src/detect-dsize.c

```diff
@@ -242,7 +242,8 @@ static DetectDsizeData *DetectDsizeParse (const char *rawstr)
         }
     }
 
-    SCLogDebug("dsize parsed successfully dsize: %"PRIu16" dsize2: %"PRIu16"",dd->dsize,dd->dsize2);
+    SCLogDebug("dsize parsed successfully dsize: %" PRIu16 " dsize2: %" PRIu16 "", dd->dsize,
+            dd->dsize2);
```

Comment:
Is this cleanup related to the patch? Should it be a part of the same commit?

Reply:
My bad for that. Clang sneaked in, after I used a `SCLogNotice` there. I'll undo that with next PR...